pkg/auth/dependency/sso/utils.go

```diff
@@ -99,7 +99,7 @@ func GetScope(scope Scope, defaultScope Scope) Scope {
 // RedirectURI generates redirect uri from URLPrefix and provider name
 func RedirectURI(URLPrefix string, providerName string) string {
 	u, _ := url.Parse(URLPrefix)
-	path := fmt.Sprintf("/auth/sso/%s/auth_handler", providerName)
+	path := fmt.Sprintf("%s/sso/%s/auth_handler", u.Path, providerName)
```

Comment:
We need to handle user input with tailing `/`. Both `http://localhost:3000/auth` and `http://localhost:3000/auth/` should be valid.